crates/atuin-client/Cargo.toml

```diff
@@ -4,7 +4,7 @@ edition = "2024"
 description = "client library for atuin"
 
 rust-version = { workspace = true }
-version = { workspace = true }
+version = "18.5.0-beta.2"
```

Comment:
(see below: if it is not easy to address the `cargo-i18n` issue then I'm fairly sure we can ignore it - either way will restore these before merge)